### PR TITLE
Adjustments refactor

### PR DESCRIPTION
**Pull request by kjharkin**

### CODE REVIEW

File: app/services/adjustmentsmanagement/AdjustmentsManagementService.scala [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+  val cdRepository: ContactDetailsRepository
+
+  def confirmAdjustment(applicationId: String, adjustmentInformation: Adjustments)
+                       (implicit hc: HeaderCarrier, rh: RequestHeader): Future[Unit] = {
+
+    val standardEventList = DataStoreEvents.ManageAdjustmentsUpdated(applicationId) ::
+      AuditEvents.AdjustmentsConfirmed(Map("applicationId" -> applicationId, "adjustments" -> adjustmentInformation.toString)) ::
+      Nil
+
+    def toEmailString(header: String, adjustmentDetail: Option[AdjustmentDetail]): String ={
```
Comment (ollyw) [inline app/services/adjustmentsmanagement/AdjustmentsManagementService.scala]: Could you pop in an extra space after `=`

File: app/services/adjustmentsmanagement/AdjustmentsManagementService.scala [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+        case Some(txt) if !txt.isEmpty => s"$header $txt"
+        case _ => ""
+      }
+    }
+
+    appRepository.find(applicationId).flatMap {
+      case Some(candidate) =>
+        cdRepository.find(candidate.userId).flatMap { cd =>
+          eventSink {
+            appRepository.confirmAdjustments(applicationId, adjustmentInformation).map{ _ =>
```
Comment (ollyw) [inline app/services/adjustmentsmanagement/AdjustmentsManagementService.scala]: could you pop an extra space between map and {?

File: app/controllers/AdjustmentsManagementController.scala [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+trait AdjustmentsManagementController extends BaseController {
+  val adjustmentsManagementService : AdjustmentsManagementService
+
+  def confirmAdjustments(applicationId: String) = Action.async(parse.json) { implicit request =>
+    withJsonBody[Adjustments] { data =>
+      adjustmentsManagementService.confirmAdjustment(applicationId, data).map { _ =>
+        Ok
+      }.recover {
+        case e: ApplicationNotFound => NotFound(s"cannot find application for user with id: ${e.id}")
```
Comment (ollyw) [inline app/controllers/AdjustmentsManagementController.scala]: it seems that the `e.id` value is actually the application id, so the message is a bit misleading

File: app/services/adjustmentsmanagement/AdjustmentsManagementService.scala (hunk 1 above)
Comment (kjharkin) [inline app/services/adjustmentsmanagement/AdjustmentsManagementService.scala]: done

Comment (kjharkin) [inline app/services/adjustmentsmanagement/AdjustmentsManagementService.scala]: Ya

File: app/controllers/AdjustmentsManagementController.scala (hunk 3 above)
Comment (kjharkin) [inline app/controllers/AdjustmentsManagementController.scala]: Changed

**Pull request merged**